test/packaging/rpm/explicit-target.py

```diff
@@ -77,7 +77,7 @@
 
 expect = """
 scons: *** Setting target is not supported for rpm.
-""" + test.python_file_line(test.workpath('SConstruct'), 12)
+""" + test.python_file_line(test.workpath('SConstruct'), 23)
```

Comment:
This has little to do with the main change, but I'm very confused by the state here: I flipped it because of 3.8 work, but the changed version doesn't come out right on any other version, fine to set it back (we don't see in CI since builders don't have rpm installed so test is skipped).  I can't figure out why Python now reports a different line number in the traceback...